src/path.rs

```diff
@@ -51,3 +54,81 @@ pub fn ensure_dir_exists(path: &Path) -> Result<()> {
     }
     Ok(())
 }
+
+pub fn warn_existing_fuel_executables() -> Result<()> {
+    let components = Components::collect_publishables()?.into_iter();
+
+    fn search_directories() -> Vec<PathBuf> {
+        if let Some(val) = env::var_os("PATH") {
+            return env::split_paths(&val).collect();
+        }
+        vec![]
+    }
+
+    let mut summary = String::new();
+
+    for c in components {
+        for e in c.executables {
+            if let Some(path) = search_directories()
+                .into_iter()
+                .map(|d| d.join(e.clone()))
+                .find(|f| is_executable(f))
+            {
+                let path = path.to_str().unwrap_or_default();
+                let mut message = String::new();
+                let fuelup_bin_dir = fuelup_bin_dir();
+                if !path.contains(fuelup_bin_dir.to_str().unwrap()) {
+                    let maybe_fuelup_executable = fuelup_bin_dir.join(&e);
+
+                    message.push_str(&format!("warning: '{e}' found in PATH at {path}."));
+
+                    if is_executable(&maybe_fuelup_executable) {
+                        message.push_str(&format!(
+                            " '{}' already installed at {} which will be overshadowed by the copy at {}.",
```

Comment:
```suggestion
                            " This will take precedence over '{}', already installed at {}. Consider uninstalling {}, or re-arranging your PATH to give fuelup priority.",
```
Maybe this is a little more consistent with the other branch's warning message? It also adds a general suggestion for uninstalling the other version or re-arranging PATH. Maybe we could add this suggestion to the other branch too?